tests/tck/features/match/OrderBy.feature

```
@@ -0,0 +1,139 @@
+      where a0 > 1
+      return v, v1
+      order by a0, name
+      """
+    Then the result should be, in order:
+      | v                                                                                                           | v1                                                                                                          |
+      | ("Manu Ginobili" :player{age: 41, name: "Manu Ginobili"})                                                   | ("Tim Duncan" :player{age: 42, name: "Tim Duncan"} :bachelor{name: "Tim Duncan", speciality: "psychology"}) |
+      | ("Tim Duncan" :player{age: 42, name: "Tim Duncan"} :bachelor{name: "Tim Duncan", speciality: "psychology"}) | ("Manu Ginobili" :player{age: 41, name: "Manu Ginobili"})                                                   |
+      | ("Tim Duncan" :player{age: 42, name: "Tim Duncan"} :bachelor{name: "Tim Duncan", speciality: "psychology"}) | ("Tony Parker" :player{age: 36, name: "Tony Parker"})                                                       |
+      | ("Tony Parker" :player{age: 36, name: "Tony Parker"})                                                       | ("Manu Ginobili" :player{age: 41, name: "Manu Ginobili"})                                                   |
+      | ("Tony Parker" :player{age: 36, name: "Tony Parker"})                                                       | ("LaMarcus Aldridge" :player{age: 33, name: "LaMarcus Aldridge"})                                           |
+      | ("Tony Parker" :player{age: 36, name: "Tony Parker"})                                                       | ("Tim Duncan" :player{age: 42, name: "Tim Duncan"} :bachelor{name: "Tim Duncan", speciality: "psychology"}) |
+
+  Scenario: topn hidden column
+    When executing query:
+      """
+      match (v:player)-->(v1:player)
+      with v, v1, v.player.name as name, count(v) as a0
+      where a0 > 0
+      return name
+      order by a0 desc, name desc
```

it seems the `a0` variable is also invisible, why not to report the semantic error?

---

It's not an error. According to cyper's standard, this case is valid. Please this the issue this PR links to, I pasted the standard there.